src/server/internalServer.cpp

```diff
@@ -406,13 +385,13 @@ bool InternalServer::start() {
 #else
   int flags = MHD_USE_POLL_INTERNALLY;
 #endif
-  if (m_verbose.load())
+  if (m_configuration.m_verbose)
     flags |= MHD_USE_DEBUG;
 
   struct sockaddr_in sockAddr;
   memset(&sockAddr, 0, sizeof(sockAddr));
   sockAddr.sin_family = AF_INET;
-  sockAddr.sin_port = htons(m_port);
+  sockAddr.sin_port = htons(m_configuration.m_port);
```

Comment:
You could have avoided the numerous changes like this one by making `InternalServer` a (private) subclass of `ServerConfiguration`.